src/internal/td.rs

```diff
@@ -15,19 +15,19 @@ pub(crate) enum TagData<'hdr> {
     Null,
 
     /// Character
-    Char(Vec<char>),
+    Char(char),
 
     /// 8-bit integer
-    Int8(Vec<i8>),
+    Int8(i8),
 
     /// 16-bit integer
-    Int16(Vec<i16>),
+    Int16(i16),
 
     /// 32-bit integer
-    Int32(Vec<i32>),
+    Int32(i32),
 
     /// 64-bit integer
-    Int64(Vec<i64>),
+    Int64(i64),
```

Comment:
I'm trying to remember why I thought these were `Vec`. I swore one of the RPM4 devs told me that, but if so I can't find the message.

Reply:
Yes, that can be important. Is like those can be arrays, but I do not see any instance in the C api for that. But my change seems indeed suspicious.

Reply:
Unfortunately I can't find the commit where I changed these from scalar values to vectors. 

Originally they looked like how you have them: https://github.com/iqlusioninc/crates/commit/ab67e78b88a6c20dc94cd9bdcb90c82d933be125#diff-8ff5175d7b1e8898d4ac2d21a9785704a370bdca9afb23e29bd2a5e7d1f805e5

At some point I changed them to `Vec`, but I can't find when that happened. In the initial import into this repo they're `Vec`:

https://github.com/rpm-software-management/librpm.rs/commit/f842876e0b4064528f983e0e52de766288f3c6cd#diff-bfa39929e8502515f224e6c3b17200e0e9ee09f5cc93267f6e2780145bc8b9a1

It might be good to open an issue or leave a comment to follow up on this.

Reply:
Opened an issue to follow up here: https://github.com/rpm-software-management/librpm.rs/issues/35